app/app.go

```diff
@@ -75,7 +81,16 @@ func addCachableHandler(e *gin.Engine, method string, endpoint string, generator
 // / This function will act as the handler for
 // / the home page
 func homeHandler(c *gin.Context, settings common.AppSettings, db database.Database) ([]byte, error) {
-	posts, err := db.GetPosts()
+	pageNumQuery := c.Param("num")
+	pageNum, err := strconv.Atoi(pageNumQuery)
+	if err != nil {
```

Comment:
Since this is also handling the `/` page (home), then this value is potentially empty and the error will be thrown. So only logging and proceeding with a nice default is probably nicer. See the comment above.